packages/devtools-network-console/src/ui/ResponseViewer/WebSocket.tsx

```diff
@@ -114,7 +126,9 @@ export default function WebSocketView(props: IWebSocketViewProps) {
                     </div>
                 </div>
             </div>
-            <div {...COMMAND_BAR_STYLE}>
+            {connected ?
+            (<div>
```

Comment:
@Rob I think you were saying this can be replaced with a fragment? prevents adding an empty wrapper div around the connection bar and monaco editor

Reply:
`<> ... </>` is the JSX for a fragment.

Reply:
thanks! updated